autoPyTorch/api/base_task.py

```diff
@@ -1237,17 +1239,35 @@ def __del__(self) -> None:
         # until the estimator is deleted
         self._backend.context.delete_directories(force=False)
 
-    @typing.no_type_check
     def get_incumbent_results(
-            self
-    ):
-        pass
+        self,
+        include_traditional: bool = False
+    ) -> Tuple[Configuration, Dict[str, Union[int, str, float]]]:
+        """
+        Get Incumbent config and the corresponding results
+        Args:
+            include_traditional: Whether to include results from tradtional pipelines
 
-    @typing.no_type_check
-    def get_incumbent_config(
-            self
-    ):
-        pass
+        Returns:
+
+        """
+        assert self.run_history is not None, "No Run History found, search has not been called."
+        if self.run_history.empty():
+            raise ValueError("Run History is empty. Something went wrong, "
+                             "smac was not able to fit any model?")
+
+        run_history_data = self.run_history.data
+        if not include_traditional:
+            # traditional classifiers have trainer_configuration in their additional info
+            run_history_data = dict(
+                filter(lambda elem: elem[1].additional_info is not None and 'trainer_configuration' not in elem[1].
```

Comment:
I am wondering if we can make this more robust. Can you add an attribute to `additional_info` so that we know for sure that this is a traditional configuration?

Reply:
makes sense. Or I can use the data origin external flag maybe